### ISSUE TITLE
Style = "centered" not supported anymore, was it removed?

### ISSUE THREAD

**Issue by lukasmartig** [davidgohel/officer#547]
When using the latest version of officer (0.6.3) in R (4.3.2) it seems that the option "centered" is not supported anymore for the body_add_plot function. For the following code and any `doc`

`doc <- doc |> body_add_plot(value = plot_instr(code = {barplot(1:5, col = 2:6)}), style = "centered")`

I get the following error: 

`could not match any style named 'centered' in c('Normal', 'heading 1', 'heading 2', 'heading 3', 'heading 4', 'heading 5', 'Title', 'Subtitle', 'No Spacing', 'Chapter', 'header', 'footer', 'HTML Preformatted', 'footnote text', 'Plain Text', 'toc 1', 'toc 2', 'toc 3', 'toc 4', 'toc 5', 'toc 6', 'toc 7', 'toc 8', 'toc 9', 'TOC Heading', 'List Paragraph', 'Balloon Text', 'Default', 'annotation text', 'annotation subject', 'caption')`

It though works with a workaround when the image is locally stored and then read into R again:

`img.file <- file.path("[my plot]"); doc |> body_add_fpar(fpar(external_img(src = img.file, height = 3.5, width = 6), fp_p = fp_par(text.align = "center")))
`

Was the option `centered` thus removed from the package?

**Comment by davidgohel** [comment]
No, style "centered" is still available in the default template provided with officer. Maybe you used another file as template and your file does not contain it

``` r
library(officer)
doc <- read_docx()
"centered" %in% styles_info(doc)$style_name
#> [1] TRUE
```

<sup>Created on 2024-01-23 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by lukasmartig** [comment]
Ah sorry, my bad. I didn't understand that the options are based on the file that one reads into R. Thought they were hard-coded as defaults in the package.

**Comment by lukasmartig** [comment]
Ah sorry, my bad. I didn't understand that the options are based on the file that one reads into R. Thought they were hard-coded as defaults in the package.

**Comment by davidgohel** [comment]
You can more about styles and template in the doc here: https://ardata-fr.github.io/officeverse/about-word-and-powerpoint-documents.html#about-word-documents
